### ISSUE TITLE
Error: ENOENT: no such file or directory, open '$HOME/.aws/credentials'

### ISSUE THREAD

**Issue by eistrati** [ddimitrioglo/aws-saml#2]
```
$ aws-saml login
fs.js:646
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: ENOENT: no such file or directory, open '$HOME/.aws/credentials'
    at Object.fs.openSync (fs.js:646:18)
    at Object.fs.readFileSync (fs.js:551:33)
    at CredentialsParser._parse ($HOME/.nvm/versions/node/v8.9.4/lib/node_modules/aws-saml/lib/credentials-parser.js:24:26)
    at new CredentialsParser ($HOME/.nvm/versions/node/v8.9.4/lib/node_modules/aws-saml/lib/credentials-parser.js:13:27)
    at login ($HOME/.nvm/versions/node/v8.9.4/lib/node_modules/aws-saml/bin/login.js:18:18)
    at Command.cli.command.description.action ($HOME/.nvm/versions/node/v8.9.4/lib/node_modules/aws-saml/bin/cli.js:45:5)
    at Command.listener ($HOME/.nvm/versions/node/v8.9.4/lib/node_modules/aws-saml/node_modules/commander/index.js:315:8)
    at emitTwo (events.js:126:13)
    at Command.emit (events.js:214:7)
    at Command.parseArgs ($HOME/.nvm/versions/node/v8.9.4/lib/node_modules/aws-saml/node_modules/commander/index.js:651:12)
```

**Comment by ddimitrioglo** [comment]
@eistrati will investigate this issue and come back with updates

**Comment by ddimitrioglo** [comment]
@eistrati fixed in version `v1.0.0`
